stetho/src/main/java/com/facebook/stetho/inspector/network/DefaultPrettyPrinterInitializer.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+public class DefaultPrettyPrinterInitializer implements AsyncPrettyPrinterInitializer {
```

Comment:
This implementation doesn't do anything at all... why not just use `null` later on instead of instantiating this? Then we can remove this class entirely